stetho_realm/src/main/java/com/uphyca/stetho_realm/RealmPeerManager.java

```diff
@@ -61,6 +61,8 @@ public void onPeerUnregistered(JsonRpcPeer peer) {
                     tableNames.add(tableName);
                 }
             }
+        } catch (Exception e) {
```

Comment:
So, if we don't catch this, then it can do one of two things: Crash the plugin, or crash the app.

I was running into app crashes before, but I think that was due to Instant Run - but honestly, I don't recall. My thought here is that at least we can log, but what would be the reason to float an exception out? Is it handled anywhere higher up?